docs/content/src/learn/guides/messaging/sending-json-data-to-a-jms-queue/README.md

```diff
@@ -0,0 +1,120 @@
+# Convert JSON to XML and upload to FTP
+
+## About 
+Ballerina is an open-source programming language that supports developers to integrate their system easily with the support of connectors. In this guide, we are mainly focusing on connecting to a ActiveMQ broker with the JMS connector. You can find other integrations modules from the [wso2-ballerina](https://github.com/wso2-ballerina) Github repository.
+
+The JMS connector created with a minimum deviation of JMS API for developers who are working with integrates messaging systems. It supports creating consumer, producer as well as listeners. You can find out more about JMS connector from [here](https://github.com/wso2-ballerina/module-jms).
+
+## What you'll build
+The Ballerina has first-class support for HTTP and implementing an HTTP service is straightforward. The caller will send JSON payload that consists of a sales order. The HTTP service read payload as text. Then with the support of JMS connector, build JMS text message and send to the ActiveMQ.
+
+## Prerequisites
+- [Java](https://www.oracle.com/technetwork/java/index.html)
+- A Text Editor or an IDE
+- Link to download Ballerina Integrator
+
+## Implementation
+The ballerina project should create for the integration use case explained above. Please follow the steps given below. You can learn about the ballerina project and module in this link.
+
+1. Create a project
+```bash
+$ ballerina new sending-json-data-to-a-jms-queue
+```
+Change directory to the sending-json-data-to-a-jms-queue.
+
+2. Add a module
+```bash
+$ ballerina add jsontojms
+```
+
+The project structure should look like below.
+```shell
+sending-json-data-to-a-jms-queue
+├── Ballerina.toml
+└── src
+    └── jsontojms
+        ├── main.bal
+        ├── Module.md
+        ├── resources
+        └── tests
+            └── resources
+```
+
+3. Write the integration
+You can open the project with VSCode. The integration implemention going to write in the `main.bal` file. 
+
+<!-- INCLUDE_CODE: src/jsontojms/main.bal -->
+
+Same as JMS API, create `jms:Connection` then `jms:Session` and finally create `jms:Destination`, `jms:MessageProducer`. In the HTTP resource, build `jms:TextMessage` getting the payload as a `string`. Next, send the message to the sales queue in the ActiveMQ. Once message send, verify the return result is an error. If it is an error, build `http:Response` as JSON payload. If it is a success, do the same. As the last step, respond to the caller with build `http:Response`.
+
+## Run the integration
+Before building the module, we have to copy the necessary ActiveMQ dependencies into the project. There are three jar files listed down below.
```

Comment:
Where do we get these files?

Reply:
Those jar files should taken from the ActiveMQ distribution. We can't give downloadable links as versions are changing based on the distribution.

Reply:
Ok. Let's just mention they have to get it from the distribution.

Reply:
done.